sdk/resourcemanager/mobilenetwork/armmobilenetwork/CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Release History
 
+## 2.0.0 (2023-01-27)
+### Breaking Changes
+
+- Function `NewPacketCoreControlPlaneClient` has been removed
+- Function `*PacketCoreControlPlaneClient.BeginCollectDiagnosticsPackage` has been removed
+- Function `*PacketCoreControlPlaneClient.BeginReinstall` has been removed
+- Function `*PacketCoreControlPlaneClient.BeginRollback` has been removed
+- Function `NewSimClient` has been removed
```

Comment:
Are all those breaking change caused by the rename of the client?
NewSimClient => NewSimsClient etc.   Can we avoid the breaking?

Reply:
Yes - this is expected - we want to get this in as soon as possible when no one is integrating with this SDK

Reply:
@lirenhe The breaking changes have been approved by Jeff in this PR: https://github.com/Azure/azure-rest-api-specs/pull/21989.